files/build_templates/init_cfg.json.j2

```diff
@@ -60,6 +60,7 @@
 {%- if include_sflow == "y" %}{% do features.append(("sflow", "disabled", true, "enabled")) %}{% endif %}
 {%- if include_macsec == "y" %}{% do features.append(("macsec", "{% if 'type' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['type'] == 'SpineRouter' and DEVICE_RUNTIME_METADATA['MACSEC_SUPPORTED'] %}enabled{% else %}disabled{% endif %}", false, "enabled")) %}{% endif %}
 {%- if include_system_gnmi == "y" %}{% do features.append(("gnmi", "enabled", true, "enabled")) %}{% endif %}
+{%- if include_bmp == "y" %}{% do features.append(("bmp", "enabled", true, "enabled")) %}{% endif %}
```

Comment:
We should not change init_cfg.json.j2, in order to enable by default in image. This enablement is controlled by golden config generation service outside sonic.